pkgs/applications/audio/raysession/default.nix

```diff
@@ -24,7 +24,7 @@ buildPythonApplication rec {
     which   # which to find lrelease.
   ];
   buildInputs = [ libjack2 ];
-  propagatedBuildInputs = [ pydbus pyliblo pyqt5 ];
+  propagatedBuildInputs = [ pydbus pyliblo pyqt5 bash ];
```

Comment:
should be in buildInputs

Reply:
Could you elaborate?

Because I assumed I understood the difference between propagated vs not, but apparently I don't. I thought propagated is for anything needed at runtime, but that's not the case because Nix is magic.

So this leaves me puzzled as to what `propagatedBuildInputs` is actually *for*. Why do some dependencies go there and others don't?